bin/release.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+set -ev
+
+# release only on merge commits or tags
+if [[ "$(git rev-list --merges HEAD^..HEAD)" || "$TRAVIS_TAG" ]]; then
+    sbt '+releaseEarly' 'project sbt-metals' '^releaseEarly'
```

Comment:
This can be automated with a `commands += Command.command("release")(s => "a" :: "b" :: s)`

Reply:
I just followed @gabro's advice here https://github.com/scalameta/metals/pull/230#issuecomment-378645212. But I'll add a command, just to keep sbt-specific things in within sbt build.